scalecube-organization/src/test/java/io/scalecube/organization/UpdateOrganizationTest.java

```diff
@@ -37,11 +37,11 @@ void updateOrganizationWithExistingOrgNameShouldFailWithNameAlreadyInUseExceptio
   }
 
   @Test
-  void updateOrganizationWithIdNotExistsShouldFailWithEntityNotFoundException() {
+  void updateOrganizationWithIdNotExistsShouldFailWithOrganizationNotFoundException() {
```

Comment:
i think we need to change the names of tests for example 

`shouldFailWithOrganizationNotFoundException`

the updateOrganizationWithIdNotExists feels very much redundant.